src/Microsoft.Health.Fhir.Shared.Client/FhirClientException.cs

```diff
@@ -13,11 +13,12 @@
 
 namespace Microsoft.Health.Fhir.Client
 {
-    public class FhirException : Exception, IDisposable
+    public class FhirClientException : Exception, IDisposable
```

Comment:
question: Did we change name to FhirClientException  because we already have FhirException under Core?

Reply:
Correct, I was getting confused by having the same exception name in two locations.